LoopKit/RemoteDataService.swift

```diff
@@ -78,6 +78,13 @@ public protocol RemoteDataService: Service {
      - Parameter completion: The completion function to call with any success or failure.
      */
     func uploadSettingsData(_ stored: [StoredSettings], completion: @escaping (_ result: Result<Bool, Error>) -> Void)
```

Comment:
This new delegate methods is so Loop can ask remote data services if they can validate a notification. More details on this flow in the Loop PR.